src/AppInstallerCLICore/Workflows/ImportExportFlow.cpp

```diff
@@ -119,6 +119,13 @@ namespace AppInstaller::CLI::Workflow
             AICLI_LOG(CLI, Info,
                 << "Installed package is available. Package Id [" << availablePackageVersion->GetProperty(PackageVersionProperty::Id) << "], Source [" << sourceDetails.Identifier << "]");
 
+            if (!availablePackageVersion->GetManifest().CurrentLocalization.Get<Manifest::Localization::Agreements>().empty())

```

Comment:
>CurrentLocalization [](http://example.com/codeflow?start=56&length=19)

any usage of CurrentLocalization requires Manifest::ApplyLocale to be called first

Reply:
I'll use DefaultLocalization here instead. Here I don't need an actual localization as we don't show the agreements, just a warning that it will require them when importing.